docs/diagrams/proxy_C4_container.puml

```diff
@@ -0,0 +1,21 @@
+@startuml proxy_c4_container
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+
+title Kentik API Proxy - Containers
+LAYOUT_LEFT_RIGHT()
+HIDE_STEREOTYPE()
+
+Boundary(shared_account, "Kentik API\nshared account") {
+    Container(user1, "User 1", "client app")
+    Container(user2, "User 2", "client app")
+    Container(user3, "User 3", "client app")
+}
+Container(proxy, "Kentik API Proxy", "GO lang", "Manages request quotas per Kentik account to mitigate throttling effects on clients")
```

Comment:
```suggestion
Container(proxy, "Kentik API Proxy", "Go", "Manages request quotas per Kentik account to mitigate throttling effects on clients")
```

Reply:
"GO lang" is more precise/searchable for people that never dealt with this technology. eg. golang.org

Reply:
When you type "go" in google, the first page displayed is https://golang.org so IMO even for non-technical readers it isn't needed.

Reply:
Of course I will not insist.

Reply:
This is because google provides targeted results based on previous searches, visited pages and cookies.
Anyway, this is more the matter of taste, please provide non-taste based suggestions :)

Reply:
Good point with targeted results.
For me "GO lang" looks like a weird typo. And I point out typos (when we have no linters for that).